snippets/gift-card-recipient-form.liquid

```diff
@@ -74,6 +74,15 @@
         {%- endif -%}
       </ul>
     </div>
+    <p
+      id="Recipient-fields-live-region-{{ section.id }}"
+      class="visually-hidden"
+      aria-live="polite"
```

Comment:
I don't believe this is needed. From the mdn docs about `role="status"`: 
![](https://screenshot.click/26-48-bkj60-j3wwk.png)